File: 2006-count-number-of-pairs-with-absolute-difference-k/2006-count-number-of-pairs-with-absolute-difference-k.py
class Solution:
    def countKDifference(self, nums: List[int], k: int) -> int:
            result = 0
            cnt = [0] * 101

            for num in nums:
                cnt[num] = cnt[num] + 1

            for i in range(k+1, 101):
                result += cnt[i] * cnt[i - k]

            return result

File: 2006-count-number-of-pairs-with-absolute-difference-k/2006-count-number-of-pairs-with-absolute-difference-k.java
class Solution {
    public int countKDifference(int[] nums, int k) {
         int cnt[]=new int[101], res = 0;
    for (int n : nums)
        ++cnt[n];
    for (int i = k + 1; i < 101; ++i)
        res += cnt[i] * cnt[i - k];
    return res;
    }
}